config/prow/config.yaml

```diff
@@ -433,6 +433,10 @@ branch-protection:
           branches:
             gh-pages:
               protect: false
+        nfs-subdir-external-provisioner:
```

Comment:
these repo lists seems to be alphabetical order by convention, could you move this down to before `node-feature-discovery`

otherwise lgtm

Reply:
important. pushed.